<launch>
    <node name = "mono_surface" pkg = "mono_surface_pub" type = "mono_surface" output = "screen">
        <!-- create aruco marker  -->
        <rosparam param="create_marker">false</rosparam>
        <!-- distortion coefficients  -->
        <rosparam param="k_">[-0.3794, 0.1686, 0.0, -0.3794, 0.1686, 0.0]</rosparam>
        <rosparam param="p_">[0.0, 0.0]</rosparam>
        <!-- camera matrix -->
        <rosparam param="fx">821.6405</rosparam>
        <rosparam param="fy">821.6648</rosparam>
        <rosparam param="cx">663.6956</rosparam>
        <rosparam param="cy">438.3716</rosparam>
    </node>
</launch>